app/Http/Controllers/Order/OrderController.php

```diff
@@ -108,13 +111,10 @@ public function getOrders(Request $request)
                 return $model->product_name;
             })
             ->addColumn('version', function ($model) {
-                return $model->product_version;
+                return $this->getLatestVersionLabel($model->product_version,$model->product);
             })
             ->addColumn('number', function ($model) {
-                return ucfirst($model->number);
-            })
-            ->addColumn('price_override', function ($model) {
-                return currency_format($model->price_override, $code = $model->currency);
+                 return $model->number.$this->getConnectionLabel($model->updated_at);
```

Comment:
please rename `getConnectionLabel ` to something meaningful

Reply:
Fixed